library/cartridge_set_scenario_steps.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pkgutil
+
+if pkgutil.find_loader('ansible.module_utils.helpers'):
+    import ansible.module_utils.helpers as helpers
+else:
+    import module_utils.helpers as helpers
+
+argument_spec = {
+    'scenario': {'required': True, 'type': 'list'},
+    'role_path': {'required': True, 'type': 'str'},
+    'custom_steps_dir': {'required': True, 'type': 'str'},
+    'custom_steps': {'required': True, 'type': 'list'},
+}
+
+
+def get_step_name_from_path(path):
+    return os.path.splitext(os.path.basename(path))[0]
+
+
+def get_steps_paths_from_dir(dir_path):
+    steps_paths = {}
+
+    if dir_path is None:
+        return steps_paths
+
+    files = os.listdir(dir_path)
+    files = filter(lambda x: x.endswith('.yml'), files)
```

Comment:
What about `.yaml`?

Reply:
Yes, it's a good idea to check for `yaml` extension